csfieldguide/templates/interactives/binary-cards.html

```diff
@@ -13,11 +13,15 @@ <h3 class="text-center mt-2">{% trans "Binary Cards" %}</h3>
           <div id="interactive-binary-cards-container" class="my-2"></div>
           <h5 id="dot-decimal-count"></h5>
 
-          <p>
-            {% trans "Flip all card sides to:" %}
+          <div class="mb-2">
+            <p class="d-inline mr-2">{% trans "Flip all card sides to:" %}</p>
             <button id="flip-to-black" class="btn btn-dark">{% trans "Black" %}</button>
             <button id="flip-to-white" class="btn btn-light">{% trans "White" %}</button>
-          </p>
+          </div>
+          <div id="cards-input" class="form-group form-inline">
+            <label for="cards-to-show" class="mr-2">{% trans "Choose how many cards to show:" %}</label>
+            <input id="cards-to-show" class="form-control" type="number" min="1" max="16" value="8"/>
```

Comment:
I think the box, and possibly the text description, should be shorter
![Uploading image.png…]()